controllers/authContorller.js

```diff
@@ -63,13 +67,88 @@ exports.signUp = async (req, res) => {
   }
 }
 
-exports.signIn = (req, res) => {
+export const signIn = (req, res) => {
   // everything is okay - send token!
   const token = getToken(req.user.id)
 
   res.send({ token })
 }
 
-exports.getCurrentUser = (req, res) => {
+export const getCurrentUser = (req, res) => {
   res.send(_.pick(req.user, ['id', 'username', 'email', 'created_at']))
 }
+
+export const requestReset = async (req, res) => {
+  const { email } = req.body
+  // make sure that user exists
+  const user = await db('users')
```

Comment:
Extract db operations to separate (User) service, import service above

Reply:
Done!